runtime/moonbase/src/precompiles.rs

```diff
@@ -232,6 +232,11 @@ type MoonbasePrecompilesAt<R> = (
 		(CallableByContract, CallableByPrecompile),
 	>,
 	PrecompileAt<AddressU64<2070>, GmpPrecompile<R>, SubcallWithMaxNesting<0>>,
+	PrecompileAt<
+		AddressU64<2071>,
```

Comment:
Please add a comment at the same place on moonriver/moonbeam to reserve the precompile address